src/briefcase/commands/new.py

```diff
@@ -117,6 +106,33 @@ def validate_app_name(self, candidate):
 
         return True
 
+    def make_class_name(self, formal_name):
+        """
+        Construct a candidate class name from a formal name.
+
+        :param formal_name: The formal name
+        :returns: The candidate app name; will appear blank if only non-latin characters
+        were entered in the formal_name
+        """
+        formal_name.title()
+        return re.sub('[^a-zA-Z]*[^a-zA-Z0-9_]+', '', formal_name)
+
+    def validate_class_name(self, candidate):
+        """
+        Determine if class name is valid
+
+        :param candidate: The candidate name
+        :returns: True. If there are any validation problems, raises ValueError
+            with a diagnostic message.
+        """
+        if not re.match('^[A-Z]+[a-zA-Z0-9_]+$', candidate):
+            raise ValueError(
+                "The class name must start with a capital letter in the CapWords format."
```

Comment:
The usual term for this is CamelCase.

Reply:
I changed the wording from CapWords to CamelCase.